cli/cmd/generate.go

```diff
@@ -38,11 +38,33 @@ var (
 		Long:    "Create IaC content for various different cloud environments and configurations",
 	}
 
+	// Azure command
+	generateAzureTfCommand = &cobra.Command{
+		Use:   "az",
+		Short: "Generate code for azure environment",
+		Long:  "Generate Terraform code for deploying into a new Azure environment.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Generate TF Code
+			cli.StartProgress("Generating Azure Terraform Code...")
+			return nil
+		},
+	}
+	// GCP command
+	generateGcpTfCommand = &cobra.Command{
+		Use:   "gcloud",
+		Short: "Generate code for gcp environment",
+		Long:  "Generate Terraform code for deploying into a new GCP environment.",
```

Comment:
Just a note here (because I messed this up originally) these `Long` descriptions are now auto-converted to docs so we'll need to build out this section (see `main` branch for example).

Reply:
will address this in the future CLI PRs. Dropped this file change from the PR.